.gitignore

```diff
@@ -0,0 +1,2 @@
+srlenium.iml
+build/
```

Comment:
read about gitignore and add more points to this list, you have example in GitHub 'Gsuite auto-test '